### ISSUE TITLE
Inconsistent definitions error

### ISSUE THREAD

**Issue by bartolsthoorn** [lanl/highs#1]
Building on Mac (but could also simply be a highs version issue?)

```bash
brew install pkg-config
brew install highs
```

```bash
$ pkg-config highs --cflags --libs
-I/opt/homebrew/Cellar/highs/1.7.2/include/highs -L/opt/homebrew/Cellar/highs/1.7.2/lib -lhighs
```

Installing highs go package:
```bash
$ go install github.com/lanl/highs
# github.com/lanl/highs
cgo: inconsistent definitions for C.kHighsObjSenseMinimize
cgo: inconsistent definitions for C.kHighsHessianFormatTriangular
cgo: inconsistent definitions for C.kHighsObjSenseMaximize
```

**Comment by spakin** [comment]
I looked into this, and it appears that it's not a HiGHS version issue but rather a GCC vs. Clang issue.  Clang, which is the default compiler on Mac, seems to be more pedantic than GCC.  I compared GCC 13.2.0 and Clang 18.1.3, both on Ubuntu 24.04.  `go build` (GCC) works, but `env CC=clang go build` (Clang) gives the same "inconsistent definitions" errors you report above.

I believe I can fix the problem by redeclaring all of the HiGHS variables and functions instead of using the HiGHS header files directly.  Stay tuned…
